nix/README.md

```diff
@@ -0,0 +1,50 @@
+# Nix build support facilities
```

Comment:
Can you add the meat of this to https://vast.io/docs/setup-vast/build#nix? In other places, we also cross-reference the main docs page from the repo.

Reply:
Can do.